tests/unit_tests/test_all_icons.py

```diff
@@ -82,17 +82,9 @@ def check_image(
         self.assertEqual(response.content_type, "image/png")
         self.assertIn('Cache-Control', response.headers)
         self.assertIn('max-age=', response.headers['Cache-Control'])
-        # reading the icon image so that we can check its size in px and average color
-        with Image.open(io.BytesIO(response.data)) as icon:
-            width, height = icon.size
-            self.assertEqual(
-                width,
-                height,
-                msg=f"Icons should be squares (wrong size of {width}px/{height}px"
-                f" for icon : {icon_name})"
-            )
-            self.assertEqual(width, expected_size)
-            if check_color:
+        if check_color:
```

Comment:
Most babs icons are neither square nor is either of their side length equal to 48. Since the front end will need to be adapted anyway to properly display such icons this check can be removed since an icon set which conforms to these rules should be check during its creation by the owner of the icon set and not by the service

Reply:
I think, we need quadratic images, otherwise the dropdown gets messed up, right @pakb @boecklic ?
So, the idea would be, to resample non-quadratic svgs and convert them to quadratic pngs, while keeping the aspect ratio.
Just did a quick test using imagemagicks convert:
`convert 030-D-EL.svg  -resize 100x100 -background white -gravity center -extent 100x100 test.png` seems to work well enough.
Probably the "100x100" needs to be adapted to something else, to make the result even "nicer".

Other option would be, to ask BABS to deliver all SVGs as square, e.g. by ensuring, that the white background canvas is quadratic, even if the colored symbol is not. No idea, if that would result in nicer converted pngs then.

Just noticed, that currently the BABS icons are transparent, while the new ones have a solid white background. Need to check with them.
Let's have a short discussion on this after the daily today. I'll need to contact BABS anyways to check on transparency, can check about the aspect ratio then, too.

Reply:
Maybe the transparent background issue is due to the conversion tool I used. But let's have a short meeting anyways to clarify transparency, size and shape of the icons anyways.